### ISSUE TITLE
control-panel schemas missing when access to remote server

### ISSUE THREAD

**Issue by pdifeo** [OpenIdentityPlatform/OpenDJ#261]
**Describe the bug**
If I access to remote server, some schemas (i.e. inetOrgPerson) is not present and I cannot edit entries.

In console I have this error
```
C:\Program Files (x86)\OpenDJ\bat>control-panel.bat -r
Exception in thread "AWT-EventQueue-0" org.forgerock.opendj.ldap.schema.UnknownSchemaElementException: No object class with name or OID "inetOrgPerson" exists in the schema
```

**Screenshots**
![image](https://user-images.githubusercontent.com/8618744/195852751-97d5a90d-250a-452c-a980-aceb5eee3870.png)

**Desktop (please complete the following information):**
 - OpenDJ 4.5.1
 - OS: Windows 10

**Comment by AlijiEmmanuel-dev** [comment]
This is probably because your server didn't load the standard LDIF files from the schema directory.
You can check the schema directory to see if the standard files are present. This is a good first step.

**Comment by vharseko** [comment]
fixed [4.6.5](https://github.com/OpenIdentityPlatform/OpenDJ/releases/tag/4.6.5)
